openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -343,6 +343,78 @@ Note: The issuer MAY decide to support both options. In which case, it is at the
 
 A Credential Format Profile for Credentials complying with IETF SD-JWT VCs [@!I-D.ietf-oauth-sd-jwt-vc] is defined in Annex A.3 of [@!OIDF.OID4VCI] and Annex A.4 of [@!OIDF.OID4VP].
 
+## SD-JWT VC Data Model (SD-JWT VCDM)
+
+SD-JWT VCDM is a credential format that is compliant to IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] that allows the usage of existing data structures represented using [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0], while enabling selective disclosure.
+
+SD-JWT VCDM credentials contain a data structure that can be processed using a JSON-LD processor after applying SD-JWT processing.
+
+When IETF SD-JWT VC is mentioned in this specification, SD-JWT VCDM defined in this section MAY be used.
+
+For backward compatibility with JWT processors, the following registered JWT claims MUST be used, instead of their respective counterpart properties in [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]:
+
+* `exp` Claim MUST represent the `expirationDate` property, encoded as a UNIX timestamp (NumericDate).
+* `iss` Claim MUST represent the `issuer` property.
```

Comment:
I would keep the original line for the following reason.

The SD-JWT VC processing is first since it deals with security-critical functions. `iss` is used by SD-JWT VC to validate the public verification key of the issuer (as per SD-JWT VC). To make sure self-declared und unverifiable values are not permitted, the linkage between the `iss` and `vcdm.issuer` (if String) /`vcdm.issuer.id` (if `vcdm.issuer` is an object) has to be kept for integrity reasons since the business logic (if `vcdm` claim is included) would rely on the SD-JWT VC verification logic that also checks the issuer.